cranelift-codegen/src/isa/x86/enc_tables.rs

```diff
@@ -1090,3 +1091,28 @@ fn convert_insertlane(
         }
     }
 }
+
+/// For SIMD negation, convert an `irsub_imm` to a `vconst + isub`
```

Comment:
```suggestion
/// For SIMD negation, convert an `ineg` to a `vconst + isub`
```

Reply:
I used fixup commits on the original commit for a cleaner history but thanks for these changes (this one and the ones below). Wish I could apply these as fixup commits and have GitHub autosquash them in...